.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
should be:
`uses: actions/checkout@163217dfcd28294438ea1c1c149cfaf66eec283e # v4.2.1`